triton-client/src/store/thunks.ts

```diff
@@ -30,9 +30,12 @@ export const fetchProjects = () => async (dispatch: AppDispatch, getState: () =>
 		dispatch(ProjectsStateActions.setLoading())
 		const projects = await apiTriton.listProjects()
 		// console.debug(`Loaded projects succesfully: ${projects}`)
-		dispatch(ProjectsStateActions.setProjects(projects))
-		dispatch(RunsStateActions.initializeRunsByProjectId(projects))
-		dispatch(DatasetsStateActions.initializeDatasetsByProjectId(projects))
+    const uniqueProjects : TritonProject[] = Array.from(new Map(projects.map((project) => {
+      return [JSON.stringify(project), project]
+    })).values())
```

Comment:
I would normally prefer using spread operation instead of `Array.from` but that would involve adding a compiler option for `downlevelIteration`.

Reply:
Can't we just rely on project name for uniqueness instead of a whole stringified json?

Reply:
hopefully ... i will change it...